### PR TITLE
Bepro 2138 apply rainbow kit instead of dappkit react

### PR DESCRIPTION
**Pull request by vhcsilva**

### CODE REVIEW

File: contexts/wagmi-provider.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+});
+
+export default function Wagmi ({
+  children
+}: WagmiProps) {
+  const cookies = parseCookies();
+
+  const config = getDefaultConfig({
+    appName: "BEPRO",
+    projectId: publicRuntimeConfig?.walletConnectProjectId || "bc2288336095f20ebf8653a1ab670566",
```
Comment (moshmage) [inline contexts/wagmi-provider.tsx:40]: default value spotted, remove for security reasons

File: x-hooks/use-dao.tsx [hunk 2]
```diff
@@ -37,29 +38,37 @@ export function useDao() {
...
-    if (selectedProvider) {
-      if ((window?.ethereum as any)?.setSelectedProvider)
-        (window.ethereum as any).setSelectedProvider(selectedProvider);
-
-      dappkitReact
-        .setProvider(selectedProvider)
-        .catch(console.debug);
-    }
+  async function connect (): Promise<string | null> {
+    const lastConnector = getLastConnector();
```
Comment (moshmage) [inline x-hooks/use-dao.tsx:55]: what happens _if i never connected before_ ?

**Review by moshmage — COMMENTED** [review]
double-check comments

File: x-hooks/use-dao.tsx (hunk 2 above)
Comment (vhcsilva) [reply x-hooks/use-dao.tsx:55]: @moshmage this `connect` is only used to auto connect the wallet if a session was already created. The first connect is made by the rainbowkit modal.

**Review by moshmage — APPROVED** [review]

**Pull request merged**